pallets/xcmp-handler/rpc/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "pallet-xcmp-handler-rpc"
+description = "RPC API to provide information about xcmp"
+version = "0.1.0"
+edition = "2021"
+authors = ["OAK Developement Team"]
+license = "GPL-3.0"
+homepage = "https://oak.tech"
+repository = 'https://github.com/OAK-Foundation/OAK-blockchain'
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"]}
+jsonrpsee = { version = "0.13.0", features = ["server", "macros"] }
+
+sp-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.23" }
+sp-blockchain = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.23" }
+sp-block-builder = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.23" }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.23" }
+sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.23" }
+frame-support= { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.23" }
+
+xcm = { git = "https://github.com/paritytech/polkadot", default-features = false, branch = "release-v0.9.23" }
+xcm-executor = { git = "https://github.com/paritytech/polkadot", default-features = false, branch = "release-v0.9.23" }
+xcm-builder = { git = "https://github.com/paritytech/polkadot", default-features = false, branch = "release-v0.9.23" }
+
+pallet-xcmp-handler-rpc-runtime-api = { path = "./runtime-api" }
+
+[features]
+default = ["std"]
+std = [
```

Comment:
why don't the other deps need to be there?

Reply:
The better question is why does this dependency need to be here.  Since this code is only executed on node, we don't need include the `std` feature (note automation time's RPC does not include the feature).  So why did I add it? `xcm-executor` is expecting the `sp_std::vec` macro to be present.

```
error: cannot find macro `vec` in this scope
   --> /Users/atomaka/.cargo/git/checkouts/polkadot-4038f27d5e4ea2e8/a7e188c/xcm/xcm-executor/src/lib.rs:177:23
    |
177 |             error_handler: Xcm(vec![]),
    |                                ^^^
    |
    = note: consider importing one of these items:
            parity_scale_codec::alloc::vec
            sp_std::vec
```

Reply:
that might be an argument to move that logic into the runtime, but I am fine either way